client/state/login/actions.js

```diff
@@ -572,3 +575,39 @@ export const logoutUser = redirectTo => ( dispatch, getState ) => {
 			return Promise.reject( error );
 		} );
 };
+
+/**
+ * Retrieves the type of authentication of the account of the specified user.
+ *
+ * @param {String} usernameOrEmail - id of the user
+ * @return {Function} a promise that will resolve once the authentication account type has been retrieved
+ */
+export const getAuthAccountType = ( usernameOrEmail ) => dispatch => {
+	dispatch( {
+		type: LOGIN_AUTH_ACCOUNT_TYPE_REQUEST,
+	} );
+
+	if ( usernameOrEmail === '' ) {
+		const error = {
+			code: 'empty_username',
+			message: translate( 'Please enter a username or email address.' ),
+			field: 'usernameOrEmail',
+		};
```

Comment:
This is a hardcoded error for now, the same one (and possibly some others) will be returned by our future API endpoint.

Reply:
I don't think we need this action creator at all, it's enough to just dispatch the action..

see: https://github.com/Automattic/wp-calypso/pull/19688/commits/475bbe6033b6ea78f70f863cb5c226bd9ea5d5ee